functional_tests/internal/api_server.go

```diff
@@ -29,13 +29,17 @@ func SetupSignalFxAPIServer(t *testing.T) {
 		ReadHeaderTimeout: 60 * time.Minute,
 	}
 
+	errCh := make(chan error)
```

Comment:
Use a channel for sending error back to the test's goroutine, as require must be used by the goroutine running the test.